v2/sarif/sarif.go

```diff
@@ -28,11 +28,17 @@ type Report struct {
 }
 
 // New Creates a new Report or returns an error
-func New(version Version) (*Report, error) {
-	schema, err := getVersionSchema(version)
-	if err != nil {
-		return nil, err
-	}
+func New(version Version, printSchema... bool) (*Report, error) {
```

Comment:
I'm happy with this in principle and it test okay - I'd preview this to be `includeSchema` as I don't think `printSchema` gets across the intent.

Otherwise I'm happy

Reply:
I agree and replaced it in the next commit